[CI] MixedClusterEsqlSpecIT test {stats.ByDateAndKeywordAndIntWithAlias SYNC} failing

**Build Scans:**
- [elasticsearch-periodic #5367 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/fqmkhazhkbxzk)
- [elasticsearch-pull-request #46392 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/bsnvooiajhg3c)
- [elasticsearch-periodic #5362 / 8.12.2_bwc](https://gradle-enterprise.elastic.co/s/4vycocdubpsog)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:mixed-cluster:v8.11.4#javaRestTest" -Dtests.class="org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT" -Dtests.method="test {dissect.OverwriteName SYNC}" -Dtests.seed=B0980C59D95C60C0 -Dtests.bwc=true -Dtests.locale=fr-CA -Dtests.timezone=Pacific/Efate -Druntime.java=23
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Bstats.ByDateAndKeywordAndIntWithAlias%20SYNC%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
org.elasticsearch.client.ResponseException: method [POST], host [http://[::1]:44971], URI [/_query?pretty=true&error_trace=true], status line [HTTP/1.1 500 Internal Server Error]
:)
úerrorúroot_causeøútypeVillegal_state_exceptionreasonàFound 3 problems
line -1:-1: Plan [ProjectExec[[full_name{r}#1948, emp_no{r}#1949, b{r}#1950, emp_no{f}#1951]]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [TopNExec[[Order[emp_no{f}#1951,ASC,LAST]],3[INTEGER],null]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [FieldExtractExec[emp_no{f}#1951]<[]>] optimized incorrectly due to duplicate output attribute emp_no{f}#1951üstack_traceäorg.elasticsearch.ElasticsearchException$1: Found 3 problems
line -1:-1: Plan [ProjectExec[[full_name{r}#1948, emp_no{r}#1949, b{r}#1950, emp_no{f}#1951]]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [TopNExec[[Order[emp_no{f}#1951,ASC,LAST]],3[INTEGER],null]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [FieldExtractExec[emp_no{f}#1951]<[]>] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
	at org.elasticsearch.server@8.11.4/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:670)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:598)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.rest.RestResponse.build(RestResponse.java:186)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:134)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:112)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:55)
	at org.elasticsearch.xpack.esql.action.EsqlResponseListener.lambda$wrapWithLogging$1(EsqlResponseListener.java:165)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListener$2.onFailure(ActionListener.java:185)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.rest.action.RestCancellableNodeClient$1.onFailure(RestCancellableNodeClient.java:96)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.client.internal.node.NodeClient$SafelyWrappedActionListener.onFailure(NodeClient.java:171)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:215)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$execute$1(ComputeService.java:135)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.acceptException(ActionListenerImplementations.java:186)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.onFailure(ActionListenerImplementations.java:191)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener.onFailure(ActionListenerImplementations.java:308)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:336)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:336)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.RefCountingListener.finish(RefCountingListener.java:127)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:117)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:69)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:116)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.Releasables$3.close(Releasables.java:142)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.RefCountingListener$1.onResponse(RefCountingListener.java:151)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.RefCountingListener$1.onResponse(RefCountingListener.java:146)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener$SuccessResult.complete(SubscribableListener.java:310)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.tryComplete(SubscribableListener.java:230)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.setResult(SubscribableListener.java:275)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.onResponse(SubscribableListener.java:173)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler.closeInternal(ExchangeSourceHandler.java:246)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:69)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$PendingInstances.finishInstance(ExchangeSourceHandler.java:267)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.onSinkComplete(ExchangeSourceHandler.java:208)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.lambda$fetchPage$1(ExchangeSourceHandler.java:165)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:177)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener$SuccessResult.complete(SubscribableListener.java:310)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.tryComplete(SubscribableListener.java:230)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.setResult(SubscribableListener.java:259)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.SubscribableListener.onResponse(SubscribableListener.java:173)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$runComputeOnRemoteNodes$8(ComputeService.java:221)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:117)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:69)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:116)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.Releasables$3.close(Releasables.java:142)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.IOUtils.close(IOUtils.java:71)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.Releasables.close(Releasables.java:33)
	at org.elasticsearch.base@8.11.4/org.elasticsearch.core.Releasables.closeExpectNoException(Releasables.java:57)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$2.run(ActionListenerImplementations.java:49)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations$RunAfterActionListener.onFailure(ActionListenerImplementations.java:271)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:336)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:54)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.handleException(TransportService.java:1719)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1436)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.transport.InboundHandler.doHandleException(InboundHandler.java:475)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.transport.InboundHandler$3.doRun(InboundHandler.java:467)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)
	at org.elasticsearch.server@8.11.4/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.IllegalStateException: Found 3 problems
line -1:-1: Plan [ProjectExec[[full_name{r}#1948, emp_no{r}#1949, b{r}#1950, emp_no{f}#1951]]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [TopNExec[[Order[emp_no{f}#1951,ASC,LAST]],3[INTEGER],null]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [FieldExtractExec[emp_no{f}#1951]<[]>] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
	at org.elasticsearch.xpack.esql.optimizer.PhysicalVerifier.verify(PhysicalVerifier.java:69)
	at org.elasticsearch.xpack.esql.optimizer.LocalPhysicalPlanOptimizer.verify(LocalPhysicalPlanOptimizer.java:47)
	at org.elasticsearch.xpack.esql.optimizer.LocalPhysicalPlanOptimizer.localOptimize(LocalPhysicalPlanOptimizer.java:43)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.lambda$localPlan$8(PlannerUtils.java:163)
	at org.elasticsearch.xpack.esql.core.tree.Node.lambda$transformUp$11(Node.java:199)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:193)
	at org.elasticsearch.xpack.esql.core.tree.Node.lambda$transformUp$10(Node.java:191)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformChildren(Node.java:212)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:191)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:199)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:152)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:141)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:134)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:508)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.lambda$runBatch$1(ComputeService.java:759)
	at org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:257)
	at org.elasticsearch.action.ActionRunnable$2.accept(ActionRunnable.java:58)
	at org.elasticsearch.action.ActionRunnable$2.accept(ActionRunnable.java:55)
	at org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:101)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$acquireSearchContexts$24(ComputeService.java:592)
	at org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:125)
	at org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:77)
	at org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:113)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.acquireSearchContexts(ComputeService.java:605)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.runBatch(ComputeService.java:756)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.start(ComputeService.java:726)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runComputeOnDataNode(ComputeService.java:795)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestHandler.messageReceived(ComputeService.java:866)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestHandler.messageReceived(ComputeService.java:843)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:76)
	at org.elasticsearch.transport.InboundHandler.doHandleRequest(InboundHandler.java:289)
	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:302)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:34)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.lang.Thread.run(Thread.java:1575)
üûùBVillegal_state_exceptionCàFound 3 problems
line -1:-1: Plan [ProjectExec[[full_name{r}#1948, emp_no{r}#1949, b{r}#1950, emp_no{f}#1951]]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [TopNExec[[Order[emp_no{f}#1951,ASC,LAST]],3[INTEGER],null]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [FieldExtractExec[emp_no{f}#1951]<[]>] optimized incorrectly due to duplicate output attribute emp_no{f}#1951üDäjava.lang.IllegalStateException: Found 3 problems
line -1:-1: Plan [ProjectExec[[full_name{r}#1948, emp_no{r}#1949, b{r}#1950, emp_no{f}#1951]]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [TopNExec[[Order[emp_no{f}#1951,ASC,LAST]],3[INTEGER],null]] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
line -1:-1: Plan [FieldExtractExec[emp_no{f}#1951]<[]>] optimized incorrectly due to duplicate output attribute emp_no{f}#1951
	at org.elasticsearch.xpack.esql.optimizer.PhysicalVerifier.verify(PhysicalVerifier.java:69)
	at org.elasticsearch.xpack.esql.optimizer.LocalPhysicalPlanOptimizer.verify(LocalPhysicalPlanOptimizer.java:47)
	at org.elasticsearch.xpack.esql.optimizer.LocalPhysicalPlanOptimizer.localOptimize(LocalPhysicalPlanOptimizer.java:43)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.lambda$localPlan$8(PlannerUtils.java:163)
	at org.elasticsearch.xpack.esql.core.tree.Node.lambda$transformUp$11(Node.java:199)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:193)
	at org.elasticsearch.xpack.esql.core.tree.Node.lambda$transformUp$10(Node.java:191)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformChildren(Node.java:212)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:191)
	at org.elasticsearch.xpack.esql.core.tree.Node.transformUp(Node.java:199)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:152)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:141)
	at org.elasticsearch.xpack.esql.planner.PlannerUtils.localPlan(PlannerUtils.java:134)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:508)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.lambda$runBatch$1(ComputeService.java:759)
	at org.elasticsearch.action.ActionListener$2.onResponse(ActionListener.java:257)
	at org.elasticsearch.action.ActionRunnable$2.accept(ActionRunnable.java:58)
	at org.elasticsearch.action.ActionRunnable$2.accept(ActionRunnable.java:55)
	at org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:101)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$acquireSearchContexts$24(ComputeService.java:592)
	at org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:125)
	at org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:77)
	at org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:113)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.acquireSearchContexts(ComputeService.java:605)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.runBatch(ComputeService.java:756)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestExecutor.start(ComputeService.java:726)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runComputeOnDataNode(ComputeService.java:795)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestHandler.messageReceived(ComputeService.java:866)
	at org.elasticsearch.xpack.esql.plugin.ComputeService$DataNodeRequestHandler.messageReceived(ComputeService.java:843)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:76)
	at org.elasticsearch.transport.InboundHandler.doHandleRequest(InboundHandler.java:289)
	at org.elasticsearch.transport.InboundHandler$1.doRun(InboundHandler.java:302)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:34)
	at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.lang.Thread.run(Thread.java:1575)
üûstatus$¨û
```

**Issue Reasons:**
- [8.x] 2 consecutive failures in step 8.11.4_bwc
- [8.x] 3 failures in test test {stats.ByDateAndKeywordAndIntWithAlias SYNC} (1.5% fail rate in 200 executions)
- [8.x] 2 failures in step 8.11.4_bwc (25.0% fail rate in 8 executions)
- [8.x] 2 failures in pipeline elasticsearch-periodic (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -428,9 +428,6 @@ tests:
 - class: org.elasticsearch.xpack.searchablesnapshots.RetrySearchIntegTests
   method: testRetryPointInTime
   issue: https://github.com/elastic/elasticsearch/issues/118514
-- class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
-  method: test {stats.ByDateAndKeywordAndIntWithAlias SYNC}
-  issue: https://github.com/elastic/elasticsearch/issues/118668
 - class: org.elasticsearch.xpack.application.OpenAiServiceUpgradeIT
   method: testOpenAiEmbeddings {upgradedNodes=1}
   issue: https://github.com/elastic/elasticsearch/issues/118156

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/dissect.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/dissect.csv-spec
@@ -223,7 +223,7 @@ null               | null      | null
 ;
 
 
-// the query is incorrectly physically plan (fails the verification) in pre-8.13.0 versions
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
 overwriteName#[skip:-8.12.99]
 from employees | sort emp_no asc | eval full_name = concat(first_name, " ", last_name) | dissect full_name "%{emp_no} %{b}" | keep full_name, emp_no, b | limit 3;
 
@@ -245,7 +245,7 @@ emp_no:integer | first_name:keyword | rest:keyword
 ;
 
 
-// the query is incorrectly physically plan (fails the verification) in pre-8.13.0 versions
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
 overwriteNameWhere#[skip:-8.12.99]
 from employees | sort emp_no asc | eval full_name = concat(first_name, " ", last_name) | dissect full_name "%{emp_no} %{b}" | where emp_no == "Bezalel" | keep full_name, emp_no, b | limit 3;
 

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/grok.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/grok.csv-spec
@@ -199,7 +199,7 @@ null               | null      | null
 ;
 
 
-// the query is incorrectly physically plan (fails the verification) in pre-8.13.0 versions
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
 overwriteName#[skip:-8.12.99]
 from employees | sort emp_no asc | eval full_name = concat(first_name, " ", last_name) | grok full_name "%{WORD:emp_no} %{WORD:b}" | keep full_name, emp_no, b | limit 3;
 
@@ -210,7 +210,7 @@ Parto Bamford     | Parto          | Bamford
 ;
 
 
-// the query is incorrectly physically plan (fails the verification) in pre-8.13.0 versions
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
 overwriteNameWhere#[skip:-8.12.99]
 from employees | sort emp_no asc | eval full_name = concat(first_name, " ", last_name) | grok full_name "%{WORD:emp_no} %{WORD:b}" | where emp_no == "Bezalel" | keep full_name, emp_no, b | limit 3;
 

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec
@@ -564,7 +564,7 @@ c:long | gender:keyword | trunk_worked_seconds:long
  0     | null           | 200000000
 ;
 
-// the query is incorrectly physically plan (fails the verification) in pre-8.13.0 versions
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
 byStringAndLongWithAlias#[skip:-8.12.99]
 FROM employees
 | EVAL trunk_worked_seconds = avg_worked_seconds / 100000000 * 100000000
@@ -720,7 +720,8 @@ c:long |           d:date         | gender:keyword | languages:integer
      2 | 1987-01-01T00:00:00.000Z | M              | 1
 ;
 
-byDateAndKeywordAndIntWithAlias
+// the query is incorrectly physically planned (fails the verification) in pre-8.13.0 versions
+byDateAndKeywordAndIntWithAlias#[skip:-8.12.99]
 from employees | eval d = date_trunc(1 year, hire_date) | rename gender as g, languages as l, emp_no as e | keep d, g, l, e | stats c = count(e) by d, g, l | sort c desc, d, l desc, g desc | limit 10;
 
 c:long |           d:date         | g:keyword | l:integer

